include/libm2k/enums.hpp

```diff
@@ -111,50 +109,42 @@ namespace libm2k {
 
 	/**
 	* @enum M2K_TRIGGER_SOURCE_ANALOG
-	* @brief The source of analog triggering.
-	*	CHANNEL_X - trigger events on analog CHANNEL_X trigger the AnalogIn interface;
-	* 	SRC_DIGITAL_IN - trigger events on the DigitalIn interface trigger the AnalogIn interface;
-	*
+	* @brief Select the source for the analog trigger
 	*/
 	enum M2K_TRIGGER_SOURCE_ANALOG {
-		CHANNEL_1 = 0,
-		CHANNEL_2 = 1,
+		CHANNEL_1 = 0, ///< CHANNEL_1 - trigger events on analog CHANNEL_1 trigger the AnalogIn interface
+		CHANNEL_2 = 1, ///< CHANNEL_2 - trigger events on analog CHANNEL_2 trigger the AnalogIn interface
 		CHANNEL_1_OR_CHANNEL_2 = 2,
 		CHANNEL_1_AND_CHANNEL_2 = 3,
 		CHANNEL_1_XOR_CHANNEL_2 = 4,
-		SRC_DIGITAL_IN = 5,
+		SRC_DIGITAL_IN = 5, ///< SRC_DIGITAL_IN - trigger events on the DigitalIn interface trigger the AnalogIn interface
 		CHANNEL_1_OR_SRC_LOGIC_ANALYZER = 6,
 		CHANNEL_2_OR_SRC_LOGIC_ANALYZER = 7,
 		CHANNEL_1_OR_CHANNEL_2_OR_SRC_LOGIC_ANALYZER = 8,
 	};
 
 
 	/**
-	 * @brief The M2K_TRIGGER_SOURCE_DIGITAL enum
-	 * 	SRC_TRIGGER_IN - trigger events on the TI(trigger in) pin trigger the DigitalIn interface;
-	 *	SRC_ANALOG_IN - trigger events on the AnalogIn interface trigger the DigitalIn interface;
-	 *	SRC_NONE - trigger events on the DigitalIn are conditioned by the internal digital trigger structure;
+	 * @enum M2K_TRIGGER_SOURCE_DIGITAL
+	 * @brief Select the source for the digital trigger
 	 */
 	enum M2K_TRIGGER_SOURCE_DIGITAL {
-		SRC_TRIGGER_IN = 0,
-		SRC_ANALOG_IN = 1,
-		SRC_NONE = 2
+		SRC_TRIGGER_IN = 0, ///< SRC_TRIGGER_IN - trigger events on the TI(trigger in) pin trigger the DigitalIn interface
+		SRC_ANALOG_IN = 1, ///< SRC_ANALOG_IN - trigger events on the AnalogIn interface trigger the DigitalIn interface
+		SRC_NONE = 2 ///< SRC_NONE - trigger events on the DigitalIn are conditioned by the internal digital trigger structure
 	};
 
 
 	/**
 	 * @enum M2K_TRIGGER_OUT_SELECT
-	 *	SELECT_TRIGGER_IN - forwards trigger events from TI pin(trigger in);
```

Comment:
curios: why not leave the docs for these values here?
would they be useful in the docs?

Reply:
Doxygen generates a nicer format if we document this using "///< " instead of just writing them down in the "@brief" or "@enum" part.

Reply:
ack
thanks for the info